CHANGELOG.rst

```diff
@@ -5,7 +5,7 @@ All notable changes to this project are documented in this file.
 
 [0.8.3-dev] in progress
 -----------------------
--
+- Remove ``ContractParameterType.Void`` (``ff``) as an optional input; improve ``get_param`` test coverage; improve ``loadcontract`` and ``gatherloadedcontractparams`` test coverage
```

Comment:
Kind of similar remark as I made in the "TX mempool PR". I'll quote
> Can we make this message a "what was solved" instead of a "how" it was solved?

The message described here is really useful for the weekly contributions gists, but from a Changelog perspective not ideal. Imagine reading this line without knowing any context other than it being `neo-python` related, would you know what it does?